tritonserver/samples/pom.xml

```diff
@@ -26,6 +26,12 @@
 	    <classifier>shaded</classifier>
         </dependency>
     </dependencies>
+    <repositories>
+        <repository>
+            <id>sonatype-nexus-snapshots</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </repository>
+    </repositories>
```

Comment:
@jbkyang-nvi removed that previously because I think she doesn't want to have users rely on external snapshots, so let's not add it again?

Reply:
> @jbkyang-nvi removed that previously because I think she doesn't want to have users rely on external snapshots, so let's not add it again?

yes